examples/alchemy-daapp/package.json

```diff
@@ -12,15 +12,18 @@
     "@chakra-ui/react": "^2.6.1",
     "@emotion/react": "^11.11.0",
     "@emotion/styled": "^11.11.0",
+    "@rainbow-me/rainbowkit": "^1.0.0",
     "@t3-oss/env-nextjs": "^0.2.1",
     "@tanstack/query-sync-storage-persister": "^4.29.7",
     "@tanstack/react-query": "^4.29.7",
     "@tanstack/react-query-persist-client": "^4.29.7",
+    "axios": "^1.4.0",
     "framer-motion": "^10.12.12",
     "next": "^13.4.1",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "react-router-dom": "^6.11.2",
+    "viem": "^0.3.30",
```

Comment:
let's update this to `latest` since it should include the fixes to signature signing